react/game/src/index.js

```diff
@@ -1,10 +1,11 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 
-import './scss/index.scss';
 import registerServiceWorker from './registerServiceWorker';
 
-import App from '~components/App'; // eslint-disable-line import/first
+import './scss/application.scss';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+import Game from '~screens/Game'; // eslint-disable-line import/first
```

Comment:
I'd recommend change `~` alias character to `@`, so u avoid linter problems like these